models/naive01.R

```diff
@@ -12,5 +15,38 @@ forecasts01=data.frame(date=forecast_date, forecastmonth=forecast_months,forecas
                          LowerPI=model01$lower[,which(model01$level==CI_level*100)], UpperPI=model01$upper[,which(model01$level==CI_level*100)])
   forecasts01[sapply(forecasts01, is.ts)] <- lapply(forecasts01[sapply(forecasts01, is.ts)],unclass)
   
-return(list(forecasts01,model01$model$aic))  
-} 
\ No newline at end of file
+  #########Include columns describing the data used in the forecast###############
+  forecasts01$fit_start_newmoon = min(abundances$newmoonnumber)
+  forecasts01$fit_end_newmoon   = max(abundances$newmoonnumber)
+  forecasts01$initial_newmoon   = max(abundances$newmoonnumber)
+  
+  aic = data.frame(date=as.Date(forecast_date), currency='abundance', model='Forecast', level=level, species='total', 
+                   aic = as.numeric(model01$model$aic), fit_start_newmoon = min(abundances$newmoonnumber),
+                    fit_end_newmoon = max(abundances$newmoonnumber), initial_newmoon = max(abundances$newmoonnumber))
+
+  return(list(forecasts01,aic))
+}
+
+#Run model on all plots and just controls
+
+#Get data
+all = read.csv("tools/rodent_all.csv")
+controls = read.csv("tools/rodent_controls.csv")
+data = yaml.load_file("tools/model.yaml")
```

Comment:
`model_metadata` is more descriptive here than `data` I think.

Reply:
:+1: